sql/moz-fx-data-shared-prod/telemetry_derived/newtab_merino_priors_v1/query.sql

```diff
@@ -1,74 +1,62 @@
-WITH
--- Define common parameters
-params AS (
-  SELECT
-    TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) AS end_timestamp,
-    TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 7 DAY AS start_timestamp
-),
-newtab_events AS (
-  SELECT
-    submission_timestamp,
-    normalized_country_code AS region,
-    event.name AS event_name,
-    SAFE_CAST(
-      mozfun.map.get_key(event.extra, 'scheduled_corpus_item_id') AS STRING
-    ) AS scheduled_corpus_item_id,
-    SAFE_CAST(mozfun.map.get_key(event.extra, 'recommended_at') AS INT64) AS recommended_at
-  FROM
-    `moz-fx-data-shared-prod.firefox_desktop.newtab`,
-    UNNEST(events) AS event,
-    params
+WITH params AS (
+  SELECT
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 1 DAY) AS end_date,
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 7 DAY) AS start_date,
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 8 DAY) AS start_date_items
+),
+corpus_items AS (
+  SELECT DISTINCT
+    corpus_item_id
+  FROM
+    `moz-fx-data-shared-prod.snowflake_migration_derived.corpus_items_current_v1`
   WHERE
-    submission_timestamp >= params.start_timestamp
-    AND submission_timestamp < params.end_timestamp
-    AND event.category = 'pocket'
-    AND event.name IN ('impression', 'click')
-    AND mozfun.map.get_key(event.extra, 'scheduled_corpus_item_id') IS NOT NULL
-    AND SAFE_CAST(mozfun.map.get_key(event.extra, 'recommended_at') AS INT64) IS NOT NULL
-    AND SAFE_CAST(mozfun.map.get_key(event.extra, 'recommended_at') AS INT64)
-    BETWEEN 946684800000
-    AND UNIX_MILLIS(
-      CURRENT_TIMESTAMP()
-    )  -- This ensures recommended_at is between Jan 1, 2000, and the current time to remain within BQ limits for dates
-),
--- Flatten events and filter relevant data
-flattened_newtab_events AS (
-  SELECT
-    *
-  FROM
-    newtab_events,
-    params
+    DATE(corpus_item_updated_at) > (SELECT start_date_items FROM params)
+),
+base AS (
+  SELECT
+    corpus_item_id,
+    country AS region,    -- rename here
+    impression_count,
+    click_count
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_derived.newtab_content_items_daily_combined_v1`
   WHERE
-    TIMESTAMP_MILLIS(recommended_at) >= params.start_timestamp
-    AND TIMESTAMP_MILLIS(recommended_at) < params.end_timestamp
+    corpus_item_id IS NOT NULL
+    AND country IS NOT NULL
```

Comment:
Do you want to ignore events without a country? I think it would bring the global count down a bit, but it may be worth checking how much.